api/app/controllers/mno_enterprise/jpi/v1/admin/sub_tenants_controller.rb

```diff
@@ -0,0 +1,63 @@
+module MnoEnterprise
+  class Jpi::V1::Admin::SubTenantsController < Jpi::V1::Admin::BaseResourceController
+
+    # GET /mnoe/jpi/v1/admin/sub_tenants
+    def index
+      # Index mode
+      @sub_tenants = MnoEnterprise::SubTenant
+      @sub_tenants = @sub_tenants.limit(params[:limit]) if params[:limit]
+      @sub_tenants = @sub_tenants.skip(params[:offset]) if params[:offset]
+      @sub_tenants = @sub_tenants.order_by(params[:order_by]) if params[:order_by]
+      @sub_tenants = @sub_tenants.where(params[:where]) if params[:where]
+      @sub_tenants = @sub_tenants.all
+      response.headers['X-Total-Count'] = @sub_tenants.metadata[:pagination][:count]
+    end
+
+    # GET /mnoe/jpi/v1/admin/sub_tenants/1
+    def show
+      @sub_tenant = MnoEnterprise::SubTenant.find(params[:id])
+      @sub_tenant_clients = @sub_tenant.clients
+      @sub_tenant_account_managers = @sub_tenant.account_managers
```

Comment:
we should probably introduce pagination for `@sub_tenant.clients` and `@sub_tenant.account_managers` at a later stage